generator-prisma-client/bun/.env.example

```
@@ -0,0 +1 @@
+DATABASE_URL=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)